src/bilder/images/consul/deploy.py

```diff
@@ -41,6 +41,7 @@
     Path("00-default.json"): ConsulConfig(
         bootstrap_expect=3,
         server=True,
+        ui=True,
```

Comment:
?

Reply:
That's so that we can use the UI for the Consul servers to identify registered services since we'll be moving away from having the Salt master running for being able to query state that way.